compass/landice/tests/ismip6_forcing/ocean_thermal/process_thermal_forcing.py

```diff
@@ -0,0 +1,319 @@
+import os
+import shutil
+import subprocess
+import xarray as xr
+from compass.landice.tests.ismip6_forcing.ocean_thermal.create_mapfile \
+    import build_mapping_file
+from mpas_tools.io import write_netcdf
+from compass.step import Step
+
+
+class ProcessThermalForcing(Step):
+    """
+    A step for creating a mesh and initial condition for dome test cases
+    """
+    def __init__(self, test_case):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.landice.tests.ismip6_forcing.ocean_thermal.
+                    OceanThermal
+            The test case this step belongs to
+        """
+        super().__init__(test_case=test_case, name='process_thermal_forcing')
+
+    def setup(self):
+        """
+        Set up this step of the test case
+        """
+        config = self.config
+        section = config['ismip6_ais']
+        base_path_ismip6 = section.get('base_path_ismip6')
+        base_path_mali = section.get('base_path_mali')
+        mali_mesh_file = section.get('mali_mesh_file')
+        period_endyear = section.get("period_endyear")
+        model = section.get("model")
+        scenario = section.get("scenario")
+
+        section = config['ismip6_ais_ocean_thermal']
+        process_obs_data = section.get("process_obs_data")
+
+        if period_endyear == "NotAvailable":
+            raise ValueError("You need to supply a user config file, which "
+                             "should contain the ismip6_ais "
+                             "section with the period_endyear option")
+        if model == "NotAvailable":
+            raise ValueError("You need to supply a user config file, which "
+                             "should contain the ismip6_ais "
+                             "section with the model option")
+        if scenario == "NotAvailable":
+            raise ValueError("You need to supply a user config file, which "
+                             "should contain the ismip6_ais "
+                             "section with the scenario option")
+
+        self.add_input_file(filename=mali_mesh_file,
+                            target=os.path.join(base_path_mali,
+                                                mali_mesh_file))
+
+        if process_obs_data == "True":
+            input_file = self._file_obs
+            output_file = f"processed_obs_TF_1995-2017_8km_x_60m.nc"
+        else:
+            input_file = self._files[period_endyear][model][scenario]
+            output_file = f"processed_TF_{model}_{scenario}_{period_endyear}.nc"
+
+        self.add_input_file(filename=os.path.basename(input_file[0]),
+                            target=os.path.join(base_path_ismip6, input_file[0]))
+        self.add_output_file(filename=output_file)
+
+    def run(self):
```

Comment:
When I run this test case, it seems like nothing happens.  It runs just a short time, there are no output files generated, and in the log I just see:
```
Loading conda environment
Done.

Loading Spack environment...
```
I'm not sure how to debug this, but I'm open to suggestions.  (cc: @xylar )